SPECS/nftables/nftables.spec

```diff
@@ -59,41 +62,39 @@ The nftables python module provides an interface to libnftables via ctypes.
 
 %build
 #./autogen.sh
-%configure --disable-silent-rules --with-xtables --with-json \
-	--enable-python --with-python-bin=%{__python3}
+%configure --disable-silent-rules --with-xtables --with-json
 %make_build
+cd py/
+%pyproject_wheel
 
 %install
 %make_install
-find %{buildroot} -type f -name "*.la" -delete -print
+find $RPM_BUILD_ROOT -name '*.la' -exec rm -f {} ';'
```

Comment:
Interesting; is `RPM_BUILD_ROOT` env var preferred over `%{buildroot}` macro var?

Reply:
and they are equivalent AFAIK other than what stage they are evaluated (i.e. macro by rpm, and env var by bash)